app/models/event.rb

```diff
@@ -10,6 +10,7 @@ class Event < ActiveRecord::Base
     authenticator_enabled: 6,
     authenticator_disabled: 7,
     account_verified: 8,
+    usps_mail_sent: 9,
```

Comment:
We need to add a translation to `config/locales/event_types/en.yml` for this, since when we create events like this we show them on the profile page (this should be visible when testing locally)